tests/Behat/Context/Reorder/ReorderContext.php

```diff
@@ -152,27 +163,19 @@ public function iShouldBeNotifiedThatNoneOfItemsFromPreviouslyPlacedOrderIsAvail
     }
 
     /**
-     * @Then /^I should have shipping address filled with (address "[^"]+", "[^"]+", "[^"]+", "[^"]+" for "[^"]+")$/
-     */
-    public function iShouldHaveTheAddressSectionFilledWithAddress(AddressInterface $address): void
-    {
-
-    }
-
-    /**
-     * @Then I should not have the shipping method section copied from order :orderNumber
+     * @Then :shippingMethod shipping method should not be selected
      */
-    public function iShouldNotHaveTheShippingMethodSectionFilledWithInformationTakenFromOrder(string $orderNumber): void
+    public function shippingMethodShouldNotBeSelected(string $shippingMethod): void
     {
-
+        assert($this->selectShippingPage->isShippingMethodSelected($shippingMethod) === false);
```

Comment:
`Assert::false($this->selectShippingPage->isShippingMethodSelected($shippingMethod))`?